README.md

```diff
@@ -582,6 +582,31 @@ Then, import it:
 Generate a nix attribute set from the opam file. This is just a Nix
 representation of the JSON produced by `opam2json`.
 
+### `queryToScopeMonorepo`
+
+```
+{ repos = ?[Repository]
+; overlays = ?[Overlay]
+; resolveArgs = ?ResolveArgs }
+→ Query
+→ Scope
+```
+
+```
+ResolveArgs :
+{ env = ?{ ${var_name} = value : String; ... }
+; with-test = ?Bool
+; with-doc = ?Bool
+; dev = ?Bool
+; depopts = ?Bool
+; best-effort = ?Bool
+}
+```
+
```

Comment:
Isn't this already defined elsewhere?

Reply:
Yep! I wasn't sure whether to include it for the sake of completeness but on second thought I'll remove it.